index.html

```diff
@@ -1,108 +1,98 @@
-			</tr>
-		</table>
-	</section>
-
-	<section class="section-2">
-		<table class="tablesection2">
-			<tr>
-				<th class="informationheadertext" colspan="4">Some random information.</th>
-			</tr>
-			<tr>
-				<td><img class="imgsection2" src="./images/html5.png" alt="html logo"></td>
-				<td><img class="imgsection2" src="./images/js.png" alt="js logo"></td>
-				<td><img class="imgsection2" src="./images/css3.png" alt="css3 logo"></td>
-				<td><img class="imgsection2" src="./images/vsc.png" alt="vsc logo"></td>
-			</tr>
-			<tr>
-				<td>this is some subtext <br> under an illustration <br> or image</td>
-				<td>this is some subtext <br> under an illustration <br> or image</td>
-				<td>this is some subtext <br> under an illustration <br> or image</td>
-				<td>this is some subtext <br> under an illustration <br> or image</td>
-			</tr>
-		</table>
-	</section>
-
-	<section class="section-3">
-		<table class="tablesection3">
-			<tr>
-				<td>
-					<p class="quotetext">This is an inspiring quote, or a testimonial from a <br>
-						customer. Maybe it's just filling up space, or maybe <br>
-						people will actually read it. Who knows? All I know <br>
-						is that it looks nice.
-					</p>
-					<p class="thor">-Thor</p>
-				</td>
-			</tr>
-			<tr>
-				<td>
-
-				</td>
-			</tr>
-		</table>
-	</section>
-
-	<section class="section-4">
-		<label class="section4label" for="">
-			<table class="section4table">
-				<tr>
-					<td class="section4bigger">Call to action! It's time!</td>
-					<td rowspan="2"><button title="button" type="button" class="btn btn-primary buttonsection4">Sign
-							up</button></td>
-				</tr>
-				<tr>
-					<td class="section4smaller">Sign up for our product by clicking that button right over there!</td>
-				</tr>
-			</table>
-		</label>
-	</section>
-
-	<footer>
-		<p>Copyright &copy;</p>
-	</footer>
-</body>
-
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Reithauser Roland, www.rolandreithauser.com">
+    <meta name="description" content="Brainnest project with CSS">
+    <meta name="generator" content="Reithauser Roland">
+    <meta name="keywords" content="HTML CSS">
+    <title>Main HTML Project CSS</title>
+    <link rel="stylesheet" href="./css/style.css" type="text/css">
+</head>
+<header>
+    <section class="headersection">
+        <div class="topnav" id="topnav">
+            <a class="headerlogo">Header Logo</a>
+            <div class="topnav-right">
+                <a href="#headerlinkone">header link one</a>
+                <a href="#header link two">header link two</a>
+                <a href="#header link three">header link three</a>
+            </div>
+        </div>
+    </section>
+</header>

```

Comment:
Hello Rolland, You did great but try putting the `Header` section inside the body. 😄